src/compiler/compile.ts

```diff
@@ -1,33 +1,32 @@
 import type { GoslingSpec, TemplateTrackDef, VisUnitApiData } from '@gosling-lang/gosling-schema';
-import type { HiGlassSpec } from '@gosling-lang/higlass-schema';
 import { traverseToFixSpecDownstream } from './spec-preprocess';
 import { replaceTrackTemplates } from '../core/utils/template';
 import { getRelativeTrackInfo, type Size, type TrackInfo } from './bounding-box';
 import type { CompleteThemeDeep } from '../core/utils/theme';
-import type { UrlToFetchOptions } from 'src/core/gosling-component';
-import { renderHiGlass as createHiGlassModels } from './create-higlass-models';
+import { collectViewsAndTracks } from './views-and-tracks';
 import { manageResponsiveSpecs } from './responsive';
-import type { IdTable } from '../api/track-and-view-ids';
 
 interface CompileResult {
-    hg: HiGlassSpec;
     size: Size;
     gs: GoslingSpec;
     tracksAndViews: VisUnitApiData[];
-    idTable: IdTable;
```

Comment:
This was also part of the HiGlass thing. Need to revisit if supporting this is necessary.